common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1955,11 +1955,11 @@ object CelebornConf extends Logging {
       .createWithDefault(16)
 
   val HDFS_DIR: OptionalConfigEntry[String] =
-    buildConf("celeborn.worker.storage.hdfs.dir")
-      .withAlternative("celeborn.storage.hdfs.dir")
-      .categories("worker")
+    buildConf("celeborn.storage.hdfs.dir")
+      .withAlternative("celeborn.worker.storage.hdfs.dir")
```

Comment:
seems we don't need this alternative, it was called `celeborn.storage.hdfs.dir` in 0.2